src/encoded/static/components/motifs.js

```diff
@@ -87,8 +87,7 @@ export class MotifElement extends React.Component {
     componentDidMount() {
         require.ensure(['d3', 'd3-sequence-logo'], (require) => {
             this.d3 = require('d3');
-            // this.sequenceLogos = logos; // This is for local development when changes are needed to d3-sequence-logo.
-            this.sequenceLogos = require('d3-sequence-logo');
+            this.sequenceLogos = logos; // This is for local development when changes are needed to d3-sequence-logo.
```

Comment:
unrelated to issues but seems like this should be put back to the require before we merge.  Do we have to build the d3-sequence-logo package?